### PR TITLE
[blog] Qdrant LangChain post

### PR DESCRIPTION
**Pull request by davidmyriel**
This post touches on the following:

1. What is Qdrant/LangChain/RAG
2. Common use cases
3. Chatbot instructions
4. Scalability
5. Additional documentation

### CODE REVIEW

File: qdrant-landing/static/blog/qdrant-and-langchain/flow-diagram.png
Comment (generall) [inline qdrant-landing/static/blog/qdrant-and-langchain/flow-diagram.png:1]: Please use the design guide-lines for images that we have

File: qdrant-landing/content/blog/using-qdrant-and-langchain.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+*Advance Data Analysis and Insights:* Sometimes you just want to browse results that are not necessarily closest, but still relevant. Qdrant's [Discovery API](https://qdrant.tech/articles/discovery-search/#discovery-search) is useful for **online stores**, where users are don't exactly know what they are looking for, but require constrained space in which a search is performed. 
+
+*Offer Content Similarity Analysis:* Ever been stuck seeing the same recommendations on your **music streaming service**? You may be held in a similarity bubble! As inputs get more complex, diversity becomes scarce, and it becomes harder to force the system to show something different. LangChain developers can use [Context API](https://qdrant.tech/articles/discovery-search/#context-search) for such cases.
```
Comment (kacperlukawski) [inline qdrant-landing/content/blog/using-qdrant-and-langchain.md:63]: Unfortunately, none of these features is implemented in Langchain. Their interfaces for vector stores are rather simple, and search is the only endpoint we use so far.

I get why we want to put it in the article, but that might be a bit confusing. How about mentioning all these use cases, without pointing to our APIs directly?

Comment (kacperlukawski) [inline qdrant-landing/content/blog/using-qdrant-and-langchain.md:63]: Langchain focuses on text, so a _music streaming service_ might not be the best analogy. How about changing it to a news recommendation?

File: qdrant-landing/content/blog/using-qdrant-and-langchain.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/O60-KuZZeQA?si=jkDsyJ52qA4ivXUy" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+## Scaling Qdrant and LangChain 
+
+If you are looking to scale up and keep the same level of performance, Qdrant and LangChain are a rock-solid combination. Getting started with both is a breeze and the [documentation](https://python.langchain.com/docs/integrations/vectorstores/qdrant) covers a broad number of cases. However, the main strength of Qdrant is that it can consistently support the user way past the prototyping and launch phases. 
+
+> *"We are all-in on performance and reliability. Every release we make Qdrant faster, more stable and cost-effective for the user. When others focus on prototyping, we are already ready for production. Very soon, our users will build successful products and go to market. At this point, I anticipate a great need for a reliable vector store. Qdrant will be there for LangChain and the entire community."*
```
Comment (kacperlukawski) [inline qdrant-landing/content/blog/using-qdrant-and-langchain.md:88]: Can we put some information about who said that? Similar to the first citation at the very beginning.

**Review by kacperlukawski — APPROVED** [review]

**Pull request merged**